R/parse-remotes.R

```diff
@@ -65,6 +93,7 @@ get_remote_types <- function(specs) {
 
   types[types == "" & grepl(standard_rx(), specs, perl = TRUE)] <- "standard"
   types[types == "" & grepl(github_rx(), specs, perl = TRUE)] <- "github"
+  types[types == "" & grepl(github_url_rx(), specs, perl = TRUE)] <- "github"
```

Comment:
I don't think it makes sense to have a separate type for this, but it makes this code sort of weird.

Reply:
It's OK.

Reply:
Alternatively we could join the two regexes in a big `(...|...)`...